src/common/droid-util.c

```diff
@@ -68,7 +68,8 @@ struct droid_quirk {
 struct droid_quirk valid_quirks[] = {
     { "input_atoi",             QUIRK_INPUT_ATOI            },
     { "set_parameters",         QUIRK_SET_PARAMETERS        },
-    { "close_input",            QUIRK_CLOSE_INPUT           }
+    { "close_input",            QUIRK_CLOSE_INPUT           },
+    { "unload_no_close",        QUIRK_UNLOAD_NO_CLOSE       }
```

Comment:
nit: ending with "," also in the final entry -> the next diff spans only one line